File: pc-square-io-separator-java/src/main/java/com/mj/pc/square/io/separator/java/constant/HtmlPage.java
package com.mj.pc.square.io.separator.java.constant;

/**
 * @author Montaser Sbaih
 * @version 1.0
 * @email montaser.jjs@gmail.com
 * @phone +962-786258874
 * @since 28-10-2022
 */

public enum HtmlPage {

    CONFIRMATION_PAGE(ResourcePath.STATIC.concat("confirmation.html"));

    private final String value;

    HtmlPage(String value) {
        this.value = value;
    }

    @Override
    public String toString() {
        return value;
    }
}

File: pc-square-io-separator-kotlin/src/main/kotlin/com/mj/pc/square/io/separator/kotlin/constant/HtmlPage.kt
package com.mj.pc.square.io.separator.kotlin.constant

/**
 * @author Montaser Sbaih
 * @version 1.0
 * @email montaser.jjs@gmail.com
 * @phone +962-786258874
 * @since 28-10-2022
 */

enum class HtmlPage(private val value: String) {

    CONFIRMATION_PAGE("${ResourcePath.STATIC}confirmation.html");

    override fun toString(): String = value
}